examples/image_classification/main.js

```diff
@@ -504,7 +367,7 @@ function main(camera) {
       }).catch((e) => {
         console.warn(`Failed to init ${utils.model._backend}, try to use WASM`);
         console.error(e);
-        showAlert(utils.model._backend);
+        showAlert(utils.model._backend, currentModel.modelName);
```

Comment:
For your information, i have totally refactored the codes for imageclassification main.js , and will also do the same thing for other examples.